x/poolmanager/taker_fee.go

```diff
@@ -13,9 +15,16 @@ import (
 )
 
 func (k Keeper) GetDefaultTakerFee(ctx sdk.Context) sdk.Dec {
-	var defaultTakerFee sdk.Dec
-	k.paramSpace.Get(ctx, types.KeyDefaultTakerFee, &defaultTakerFee)
-	return defaultTakerFee
+	defaultTakerFeeBz := k.paramSpace.GetRaw(ctx, types.KeyDefaultTakerFee)
+	if !bytes.Equal(defaultTakerFeeBz, k.defaultTakerFeeBz) {
```

Comment:
ditto here for the value for `k.defaultTakerFeeBz`

Reply:
Intended to be nil!

Reply:
Oh yes, whoops